main.c

```diff
@@ -254,6 +253,52 @@ int main(int argc, char **argv)
             {
                 connected = 1;
             }
+        }else if(connected == 4)
+        {
+            Entry* glycemia = getGlycemiaDataFromDB(user_id);
+            printf("\n---------- Logs from Date -----------:\n");
+            printf("1. See your glycemia logs for a specific date\n");
+            printf("2. See your glycemia before a specific date\n");
+            printf("3. See your glycemia after a specific date\n");
+            printf("4. See your glycemia between two dates\n"); 
+            printf("0. Exit Logs from Date\n");
+            scanf(" %c", &choice);
+            
+            if(choice == '1')
+            {
+                printf("\nPlease enter the date you want to see the logs for (dd/mm/yyyy):\n");
+                scanf("%s", &date);
+                //show glycemia logs for a specific date
+                showEntriesForDate(glycemia, date);
```

Comment:
Need to check if the user input isn't dangerous and is of correct format.
Risk of SQL Injection ?

Reply:
no need cause of the size of date :)